test/bench/bench.go

```diff
@@ -178,6 +192,53 @@ var (
 	}
 )
 
+var (
+	ReferenceCheckerCases = []*EvalCase{
```

Comment:
nit: Consider renaming this to ReferenceDynamicEnvCases and add a comment to the exposed variable?